.ebextensions/99datadog.config

```diff
@@ -0,0 +1,95 @@
+# Reference: https://docs.datadoghq.com/integrations/amazon_elasticbeanstalk/?tab=singlecontainer
+# .ebextensions/99datadog.config
+option_settings:
+    - namespace:  aws:elasticbeanstalk:application:environment
+      option_name:  DD_AGENT_VERSION
+      value:  "" # For example, "7.21.1". Leave empty to install the latest version. Only Agent 7 is supported.
+files:
+    "/configure_datadog_yaml.sh":
+        mode: "000700"
+        owner: root
+        group: root
+        content: |
+            #!/bin/bash
+
+            sed 's/api_key:.*/api_key: '$DD_API_KEY'/' /etc/datadog-agent/datadog.yaml.example > /etc/datadog-agent/datadog.yaml
+            # Use the following command if you're in the EU. Replace datadoghq.eu with another region if in a different region.
```

Comment:
You would need to enable non local traffic since the agent is on host and the docker container is considered a remote. See AskGov PR.

Reply:
Hmm, weird, appears to be working. I guess I'll make changes on AskGov and observe results.

Reply:
would that explain the orange hexagons? i think datadog is also not picking up the correct `env`

Reply:
Actually, yes that would.

Reply:
btw, I spoke to @yong-jie yesterday too; coming round to your POV that it's probably better to migrate to Amazon Linux 2 first before pushing this out

Reply:
let me test enabling non-local traffic and configuring the hostname; perhaps it might work or be good enough